ios/CodePush/CodePush.m

```diff
@@ -458,11 +459,33 @@ - (void)savePendingUpdate:(NSString *)packageHash
                            rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSError *error;
+    if (isRunningBinaryVersion) {
+        // isRunningBinaryVersion will not get set to "YES" if running against the packager.
+        NSString *binaryHash = [CodePushUpdateUtils getHashForBinaryContents:[CodePush binaryBundleURL] error:&error];
```

Comment:
Could we save this hash somewhere so that at most, we only have to compute the hash on the first run after a binary installation? I'd prefer to keep app start as light as possible.